packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -214,6 +215,11 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
     ...props,
   };
 
+  // TODO AS-136-submit-content-modal update with value from API after extension adoption
+  // will use sourceRequestAvailability query when adoption is reached on extension
+  const REPUTATION_THRESHOLD = 250;
+  const isEnabled = isLoggedIn && user?.reputation >= REPUTATION_THRESHOLD;
```

Comment:
Side note, non blocking.
Maybe we should start working with 1 default const file for magic numbers? (wdyt?)
Want to avoid someone saying oh let's adjust treshold to 300 and forgetting all the placements.

Reply:
This will be removed once we hook up API implementation which will tell if it is allowed. Then we will be able to change the threshold on API without frontend changes. This is just for back compat until extension adoption is reached.

Reply:
It will use query from https://github.com/dailydotdev/daily-api/pull/1722